src/Notifications/Notifiable.php

```diff
@@ -18,6 +18,11 @@ public function routeNotificationForSlack(): string
         return config('backup.notifications.slack.webhook_url');
     }
 
+    public function routeNotificationForDiscord(): string
```

Comment:
I think that this could/should be renamed to `routeNotificationForDiscordWebhook()` - and therefore also the notification channel name!? 🤔
This will ensure that there's no conflict.

https://github.com/laravel-notification-channels/discord#usage